### PR TITLE
UND-441][DEVELOP]- Single content card feedback issues

### PR DESCRIPTION
**Pull request by neelesh-tippe**
[UND-441][DEVELOP]- Single content card feedback issues

### CODE REVIEW

**Comment by sumit3021kumar** [comment]
Hey @neelesh-tippe  Please resolve the conflict

File: stories/Organism/Blocks/ContentCardWithAndWithoutImage/ContentCardWithAndWithoutImage.jsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ export const ContentCardWithAndWithoutImage = ({
                 </div>
                 <div className="content-card__caption">
                   <Heading type="5" label={item.contentname} />
-                  <Ctalink label={item.button} Type={item.btnType} />
+                  <span className="cta__link cta--space">{item.button}</span>
```
Comment (sumit3021kumar) [inline stories/Organism/Blocks/ContentCardWithAndWithoutImage/ContentCardWithAndWithoutImage.jsx:29]: Hey @neelesh-tippe  can we replace this mark with cta component eg :
`<Ctalink label={item.button} Type="Space" Emphasize="false"/>`

File: stories/Organism/Blocks/ContentCardWithAndWithoutImage/ContentCardWithAndWithoutImage.jsx [hunk 2]
```diff
@@ -57,7 +57,7 @@ export const ContentCardWithAndWithoutImage = ({
                     : (
                       <></>
                     )}
-                  <Ctalink label={item.button} Type={item.btnType} />
+                  <span className="cta__link cta--space">{item.button}</span>
```
Comment (sumit3021kumar) [inline stories/Organism/Blocks/ContentCardWithAndWithoutImage/ContentCardWithAndWithoutImage.jsx:60]: Hey @neelesh-tippe  can we replace this mark with cta component eg :
`<Ctalink label={item.button} Type="Space" Emphasize="false"/>`

Comment (neelesh-tippe) [reply stories/Organism/Blocks/ContentCardWithAndWithoutImage/ContentCardWithAndWithoutImage.jsx]: Done, can you please review and merge this PR

File: stories/Organism/Blocks/ContentCardWithAndWithoutImage/ContentCardWithAndWithoutImage.jsx (hunk 1 above)
Comment (neelesh-tippe) [reply stories/Organism/Blocks/ContentCardWithAndWithoutImage/ContentCardWithAndWithoutImage.jsx]: Done

**Pull request closed without merging**